### PR TITLE
Graphite scaler docs

### PR DESCRIPTION
**Pull request by bpinske**
_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #1628

### CODE REVIEW

**Review by tomkerkhove — APPROVED** [review]

**Review by zroubalik — COMMENTED** [review]
Could you please update the file, so for example the Parameter list items description is more verbose.

Add more details to the `metricName` (as it is done for other scalers). More examples for `queryTime` would be great and does the value needs to be negative?

**Comment by zroubalik** [comment]
And I don't see a section on `auth`

**Comment by bpinske** [comment]
> I don't see a section on `auth`
> Could you please update the file, so for example the Parameter list items description is more verbose.

> Add more details to the `metricName` (as it is done for other scalers). More examples for `queryTime` would be great and does the value needs to be negative?

All comments addressed. Verbosity is comparable to the Prometheus scaler.

Yes, the value does need to be a negative relative lookback time to be useful.
https://graphite-api.readthedocs.io/en/latest/api.html#from-until

File: content/docs/2.5/scalers/graphite.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+This specification describes the `graphite` trigger that scales based on metrics in Graphite.
+
+```yaml
+triggers:
+- type: graphite
+  metadata:
+    # Required
+    serverAddress: http://<graphite-host>:81
+    metricName: request-count # Note: metric sort name in query
```
Comment (zroubalik) [inline content/docs/2.5/scalers/graphite.md:20]:
```suggestion
    metricName: request-count # Note: name to identify the Metric
```

File: content/docs/2.5/scalers/graphite.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    threshold: '100'
+    queryTime: '-10Minutes' # Note: Query time in from argv Seconds/Minutes/Hours
+```
+**Parameter list:**
+
+- `serverAddress` - Address of Graphite
+- `metricName` - Name to identify the Metric in the external.metrics.k8s.io API. If using more than one trigger it is required that all `metricName`(s) be unique
+- `query` - Query to run
+- `threshold` - Value to start scaling for
+- `queryTime` - Relative time range to execute query against.
```
Comment (zroubalik) [inline content/docs/2.5/scalers/graphite.md:31]: Might be nice to add link to https://graphite-api.readthedocs.io/en/latest/api.html#from-until

**Review by zroubalik — APPROVED** [review]

**Pull request merged**